src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestDetailsView.tsx

```diff
@@ -93,101 +100,136 @@ const ChangeRequestDetailsView: React.FC<ChangeRequestDetailsProps> = ({
-  if (!project || isLoading) return <LoadingIndicator />;
+  if (isErrorAllUsers) return <ErrorPage message={errorAllUsers?.message} />;
+  if (!project || isLoading || isLoadingAllUsers || !users) return <LoadingIndicator />;
+
   const { name: projectName } = project;
 
-  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
-    setAnchorEl(event.currentTarget);
+  const handleRequestReviewerClick = async () => {
+    if (reviewerIds.length === 0) return;
+    try {
+      await requestCRReview({ userIds: reviewerIds });
+    } catch (e) {
+      if (e instanceof Error) {
+        toast.error(e.message);
+      }
+    }
   };
 
-  const handleDropdownClose = () => {
-    setAnchorEl(null);
-  };
+  const isRequestAllowed = changeRequest.submitter.userId === currentUser.userId;
 
   const unreviewedActionsDropdown = (
+    <ActionsMenu
+      style={{ mt: '10px', float: 'right' }}
+      buttons={[
+        {
+          title: 'Review',
+          onClick: handleReviewOpen,
+          disabled: !isUserAllowedToReview,
+          icon: <EditIcon fontSize="small" />
+        },
+        {
+          title: 'Delete',
+          onClick: handleDeleteOpen,
+          disabled: !isUserAllowedToDelete,
+          icon: <DeleteIcon fontSize="small" />
+        }
+      ]}
+    />
+  );
+
+  const requestReviewerDropdown = (
     <div>
-      <NERButton
-        endIcon={<ArrowDropDownIcon style={{ fontSize: 28 }} />}
-        variant="contained"
-        id="unreviewed-cr-actions-dropdown"
-        onClick={handleClick}
-      >
-        Actions
-      </NERButton>
-      <Menu open={dropdownOpen} anchorEl={anchorEl} onClose={handleDropdownClose}>
-        <MenuItem onClick={handleReviewOpen} disabled={!isUserAllowedToReview}>
-          Review
-        </MenuItem>
-        <Divider />
-        <MenuItem disabled={!isUserAllowedToDelete} onClick={handleDeleteOpen}>
-          Delete
-        </MenuItem>
-      </Menu>
+      <Autocomplete
+        isOptionEqualToValue={(option, value) => option.id === value.id}
+        limitTags={1}
+        disableCloseOnSelect
+        multiple
+        options={users.filter((user) => isLeadership(user.role)).map(userToAutocompleteOption)}
+        getOptionLabel={(option) => option.label}
+        onChange={(_, values) => setReviewerIds(values.map((value) => value.id))}
+        renderTags={() => null}
+        renderOption={(props, option, { selected }) => (
+          <li {...props}>
+            <Checkbox
+              icon={<CheckBoxOutlineBlankIcon />}
+              checkedIcon={<CheckBoxIcon />}
+              style={{ marginRight: 8 }}
+              checked={selected}
+            />
+            {option.label}
+          </li>
+        )}
+        renderInput={(params) => (
+          <TextField {...params} variant="standard" placeholder={`${reviewerIds.length} Reviewers Selected`} />
+        )}
+      />
+
+      <Grid container spacing={1}>
```

Comment:
this grid nonsense is way to complicated for what's being done here. We overuse it in the frontend rn, so I can see why you ended up using it. If you delete all the grid code, and just do this, it should work better and be less code. I'd read into how flex boxes work, the gap prop, and the justifyContent prop if you're curious why this works:

```
      <Box display="flex" flexDirection="row" gap="10px" justifyContent="right">
        <NERButton
          sx={{ mt: '10px', float: 'right' }}
          variant="contained"
          disabled={!isRequestAllowed}
          onClick={handleRequestReviewerClick}
        >
          Request Review
        </NERButton>
        {unreviewedActionsDropdown}
      </Box>
```